_tests_/ports.test.js

```diff
@@ -0,0 +1,36 @@
+const Port = require('../src/ports.js');
```

Comment:
This seems to be working, but it has problems with indentation and syntax throughout:

1. Indentation is only 2 spaces whereas the rest of the PR is 4,
2. Several missing semi colons at line ends on variable declaration,
3. Closing brackets on functions not being at the same level as their opening brackets